infrastructure/buckets/outputs.tf

```diff
@@ -104,10 +104,16 @@ output "object_references" {
       bucket : aws_s3_bucket_object.change_company_publicly_function_object.bucket
      }
 
-     "get_all_public_companies_function_bucket" : {
+    "get_all_public_companies_function_bucket" : {
       etag : aws_s3_bucket_object.get_all_public_companies_function_object.etag,
       key : aws_s3_bucket_object.get_all_public_companies_function_object.key,
       bucket : aws_s3_bucket_object.get_all_public_companies_function_object.bucket
     }
+
+    "upload_data_s3_trigger_function_bucket" : {
```

Comment:
Can we change the name?
The lambda is just a connection with API Gateway for and endpoint call, "trigger" can be confused name for the actual functionality.
The lambda that starts running the job in glue is a trigger.

Reply:
resources were changed to 'upload_file_s3' and endpoint is now 'upload_file'